PubMatic/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 4.8.0.0
+* Certified with PubMatic SDK 4.8.0.
+* Added support for PubMatic OpenWrap SDK Native Ads.
```

Comment:
IMO, no need to mention `PubMatic OpenWrap SDK` , as this adapter and changelog is for `PubMatic OpenWrap SDK` only.